### PR TITLE
Build with .NET 6

### PR DESCRIPTION
**Pull request by jimmyca15**
This PR updates the PR build pipeline to include .NET 6

### CODE REVIEW

File: build/install-dotnet.ps1 [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Installs .NET Core 2.1 and .NET 6 for CI/CD environment
+# see: https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-install-script#examples
+
+[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12;
+
+&([scriptblock]::Create((Invoke-WebRequest -UseBasicParsing 'https://dot.net/v1/dotnet-install.ps1'))) -Version 2.1.816
```
Comment (pratiksanglikar) [inline build/install-dotnet.ps1:6]: > -Version 2.1.816

Why do we need version `2.1.816`?

Comment (jimmyca15) [reply build/install-dotnet.ps1:6]: Tests still build with .NET Core 2.1 for the time being to make sure we exercise the netstandard 2.0 build.

File: .pipelines/pipeline.user.windows.yml [hunk 2]
```diff
@@ -7,16 +7,21 @@ environment:
   
 restore:
   commands:
+    - !!defaultcommand
+      name: 'Install .NET Core 6'
```
[the remark this answers is not in the document]
Comment (jimmyca15) [reply .pipelines/pipeline.user.windows.yml:11]: You're right, thanks, will update.

[the remark this answers is not in the document]
Comment (jimmyca15) [reply .pipelines/pipeline.user.windows.yml]: Updated.

**Review by pratiksanglikar — APPROVED** [review]

**Pull request merged**